virt/upgrade/_attributes

```diff
@@ -0,0 +1 @@
+../../_attributes/
```

Comment:
Not sure if you will need to re-do anything, link-wise, if you change the dir name to "updating." Maybe not?